vector/src/main/res/layout/item_tchap_room.xml

```diff
@@ -66,10 +66,12 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:contentDescription="@string/room_encrypted"
+        android:paddingTop="6dp"
         app:layout_constraintCircle="@id/roomAvatarContainer"
         app:layout_constraintCircleAngle="45"
         app:layout_constraintCircleRadius="30dp"
-        tools:ignore="MissingConstraints" />
+        app:layout_constraintEnd_toEndOf="@id/roomAvatarContainer"
+        app:layout_constraintTop_toTopOf="parent" />
```

Comment:
You should not add these constraints, there is already a circle constraint with a radius and an angle. If the padding does not work as expected, you may probably adjust the circle properties